app/grandchallenge/products/sitemaps.py

```diff
@@ -22,3 +23,14 @@ def items(self):
 
     def lastmod(self, item: Product):
         return item.modified
+
+
+class PostsSitemap(SubdomainSitemap):
```

Comment:
This class doesn't do anything as it's not registered, and even if it was it just duplicates what is in `grandchallenge.blogs.sitemaps.PostsSitemap`. It can be removed.

Reply:
removed